pyinterprod/interpro/contrib/pfam.py

```diff
@@ -109,7 +109,21 @@ def _repl(match: re.Match) -> str:
 
         return f"[{','.join(refs)}]"
 
-    return re.sub(r"\[([\d\s,]+)]", _repl, text)
+    return re.sub(r"\[([\d\s,-]+)]", _repl, text)
+
+
+def _expand_range(s: str) -> list[int]:
+    r = []
+    for i in s.split(','):
+        values = i.split("-")
+        if len(values) == 1:
+            # single reference number
+            r.append(int(i))
+        else:
+            # range of reference numbers, e.g. 1-4
+            low, high = map(int, values)
+            r += list(range(low, high + 1))
+    return(r)
```

Comment:
Blank line before `return`, and also `return` is a statement and doesn't need parentheses, unless you want to return a tuple.

```suggestion

    return r
```